src/main/java/io/github/hildi/can/service/StandardJavaSerializationService.java

```diff
@@ -0,0 +1,48 @@
+package io.github.hildi.can.service;
+
+import io.github.hildi.can.exceptions.DeserializeException;
+import io.github.hildi.can.exceptions.FileIsNotExistsException;
+import io.github.hildi.can.exceptions.NotWritableFileException;
+import io.github.hildi.can.model.User;
+
+import java.io.*;
+
+/**
+ * Created by Serhii Hildi on 15.03.19.
+ */
+public class StandardJavaSerializationService implements SerializationService {
+
+    private User user;
+
+    @Override
+    public void serialize(User user, File file) {
+
+        if (!file.exists()) {
+            throw new FileIsNotExistsException("We cannot write data at file. File is not exists.");
```

Comment:
1) avoid personate error messages
`we cannot do something` - bad
`failed to serialize data, reason: file doesn't exist` - good
2) actually, there is no useful information in that error message, it's already clear from the name of exception that file doesn't exist. it's better to give some additional helpful information, i.e. file name.